### ISSUE TITLE
[Mac] IsPlaying, IsPaused, IsSeeking incorrectly lag behind the true playback state

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#687]
On Mac AVPro 2.0.9, but not on Windows, the MediaPlayer.Control methods IsPlaying, IsPaused, and IsSeeking are all incorrect right after a Play, Pause, or Seek operation respectively.

To reproduce, in Demo_MediaPlayer, edit MediaPlayerUI.Update to add this:

```
if (Input.GetKeyDown(KeyCode.A)) {
	Debug.Log("Pre-play: IsPlaying " + _mediaPlayer.Control.IsPlaying());
	_mediaPlayer.Play();
	Debug.Log("Post-play: IsPlaying " + _mediaPlayer.Control.IsPlaying());
}
if (Input.GetKeyDown(KeyCode.S)) {
	Debug.Log("Pre-pause: IsPaused " + _mediaPlayer.Control.IsPaused());
	_mediaPlayer.Pause();
	Debug.Log("Post-pause: IsPaused " + _mediaPlayer.Control.IsPaused());
}
if (Input.GetKeyDown(KeyCode.D)) {
	Debug.Log("Pre-seek: IsSeeking " + _mediaPlayer.Control.IsSeeking());
	_mediaPlayer.Control.Seek(0.01);
	Debug.Log("Post-seek: IsSeeking " + _mediaPlayer.Control.IsSeeking());
}
```

***

First test: IsPlaying.
Launch the demo, then click to pause the default playback. Press A to play.

Expected result, and actual result on Windows:
```
Pre-play: IsPlaying false
Post-play: IsPlaying true
```

Actual result on Mac:
```
Pre-play: IsPlaying false
Post-play: IsPlaying false
```

***

Second test: IsPaused.
Launch the demo, then press S to pause.

Expected result, and actual result on Windows:
```
Pre-pause: IsPaused false
Post-pause: IsPaused true
```

Actual result on Mac:
```
Pre-pause: IsPlaying false
Post-pause: IsPlaying false
```

***

Third test: IsSeeking.
Launch the demo, then press D to seek.

Expected result, and actual result on Windows:
```
Pre-seek: IsSeeking false
Post-seek: IsSeeking true
```

Actual result on Mac:
```
Pre-seek: IsSeeking false
Post-seek: IsSeeking false
```

**Comment by AndrewRH** [comment]
@TheloTheGreat 
2.0.10 has just been release - please let us know whether this fixes the issue for you.

**Comment by TheloTheGreat** [comment]
2.0.10 does fix this issue, thanks.

However, it does not fix the associated issue #685, which was closed as being the same as this issue. I can't reopen #685 myself to flag it as fix-failed.
